Client/Frontend/Settings/CustomSearchViewController.swift

```diff
@@ -50,9 +49,11 @@ class CustomSearchViewController: SettingsTableViewController {
         super.viewDidLoad()
         title = .SettingsAddCustomEngineTitle
         view.addSubview(spinnerView)
-        spinnerView.snp.makeConstraints { make in
-            make.center.equalTo(self.view.snp.center)
-        }
+        spinnerView.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
Instead of this, we should, in the spinner view var, covert it to a `.build { spinner in ...}` pattern that we use throughout the application. Examples might be found all over, but `OnboardingCardViewController` would be a good file to look at.

Reply:
Thank you for suggestion. Please see [26549b3](https://github.com/mozilla-mobile/firefox-ios/pull/17089/commits/26549b325f8e4f0b28457c4027dff517fae17c22)